backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/interceptor/C_Order.java

```diff
@@ -54,18 +59,32 @@ public C_Order(@NonNull final ModularContractService contractService)
 	@DocValidate(timings = ModelValidator.TIMING_AFTER_COMPLETE)
 	void afterComplete(@NonNull final I_C_Order orderRecord)
 	{
-		contractService.invokeWithModel(orderRecord, COMPLETED);
+		invokeHandlerForEachLine(orderRecord, COMPLETED);
+	}
+
+	@DocValidate(timings = ModelValidator.TIMING_AFTER_VOID)
+	void afterVoid(@NonNull final I_C_Order orderRecord)
+	{
+		invokeHandlerForEachLine(orderRecord, VOIDED);
 	}
 
 	@DocValidate(timings = { ModelValidator.TIMING_AFTER_REVERSEACCRUAL, ModelValidator.TIMING_AFTER_REVERSECORRECT })
 	void afterReverse(@NonNull final I_C_Order orderRecord)
 	{
-		contractService.invokeWithModel(orderRecord, REVERSED);
+		invokeHandlerForEachLine(orderRecord, REVERSED);
```

Comment:
An order cannot be reversed. Maybe we should throw Unsupported Operation Eception ?